generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -370,6 +370,13 @@ kafka:
     key.serializer: org.apache.kafka.common.serialization.StringSerializer
     value.serializer: org.apache.kafka.common.serialization.StringSerializer
 <%_ } _%>
+
+#  Uncomment to override generated content security policy
+#    security:
+#       headers:
+#           content-security-policy:
```

Comment:
Why not generate the default here, such the default/fallback is not in code?

Reply:
Also good solution.  And would be more visible to user.  As for dedicated jhipster  property - does it not mandate change on jhipster library classes?

Or maybe put it as configuration into yo-rc?

Reply:
I would not put it into yo-rc personally as that would make it hard to grasp where the property comes from and how to update.